packages/graph/lib/graph.ts

```diff
@@ -80,4 +91,92 @@ export class Graph {
         // detach from node 2
         n2.unlinkChannel(channel);
     }
+
+    public dijkstra(src: Node, dest: Node, amnt: bigint) {
+        // Here we will be first computing route using reverse dijkstra that is
+        // from dest --> src and return route from src --> dest
+
+        // Convert it to corresponding node_id
+        const str_id = dest.nodeId.toString("hex");
+
+        // Distances will store the base_fee required to traverse from the src
+        let distances = {},
```

Comment:
Language nit: use maps over objects.  Even though JS/TS allows property mutation, memory management is most effective when types remain static.  As such the Set/Map types are better suited for that type of stuff.